openipam/api/views/network.py

```diff
@@ -313,10 +314,8 @@ def post(self, request, format=None, **kwargs):
 
 class NetworkList(generics.ListAPIView):
     permission_classes = (permissions.IsAuthenticated,)
-    queryset = Network.objects.all()
+    queryset = Network.objects.all().annotate(network_in=F("network"))
```

Comment:
You shouldn't need to annotate the queryset with a duplicate field like this just to add multiple filters to the field. You should be able to specify the model field used by the filter in the filter field's constructor.